### DISCUSSION TITLE
需要补档的游戏

### DISCUSSION THREAD

**Discussion by hyh19962008** [Gamuxorg/bbs#218] [投递和请求游戏]
下载链接清理完了，有空的认领一下……

- [x] 公路救赎 Road Redemption 中文版
- [ ] AER：古老的回忆
- [x] 英雄无敌3[死亡阴影]vcmi版
- [x] 乐园
- [ ] 海岛大亨
- [ ] 深海远航
- [x] 黑客网络，运行有bug
- [x] 盐和避难所，运行有bug 
- [ ] AER：古老的回忆
- [ ] FEZ

**Comment by shouhuanxiaoji** [comment]
盐和避难所已在manjaro测试

**Comment by shouhuanxiaoji** [comment]
海岛大亨是几？

**Comment by shouhuanxiaoji** [comment]
乐园已经补档
